src/main/java/org/mockito/stubbing/Answer1.java

```diff
@@ -14,11 +14,13 @@
  * Example of stubbing a mock with this custom answer:
  *
  * <pre class="code"><code class="java">
- * when(mock.someMethod(anyString())).thenAnswer(new Answer&lt;Integer, String&gt;() {
- *     Integer answer(String arg) {
- *         return arg.length();
- *     }
- * });
+ * import static org.mockito.AdditionalAnswers.answer;
+ *
+ * when(mock.someMethod(anyString())).then(answer(
+ *     new Answer1&lt;Integer, String&gt;() {
+ *         public Integer answer(String arg) {
+ *             return arg.length();
+ *         }}));
```

Comment:
Please put back the original location of the brackets. This is also true for all other changes